src/main/java/pl/agh/edu/master_diet/controller/GamificationController.java

```diff
@@ -0,0 +1,24 @@
+package pl.agh.edu.master_diet.controller;
+
+import javassist.NotFoundException;
+import lombok.AllArgsConstructor;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import pl.agh.edu.master_diet.core.model.rest.gamification.PointsRequest;
+import pl.agh.edu.master_diet.service.GamificationService;
+
+@RestController
+@RequestMapping("/gamification")
+@AllArgsConstructor(onConstructor = @__(@Autowired))
+public class GamificationController {
+
+    private final GamificationService gamificationService;
+
+    @PostMapping
+    public void addPoints(@RequestBody PointsRequest request) throws NotFoundException {
```

Comment:
User can invoke this endpoint manually and add points with no limit

Reply:
you are right it is unnecessary